<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LAS TRX</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 56, 70);
color: rgb(143, 240, 164)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_input">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(36, 31, 49);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_input">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Origin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_input">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_input_file" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_input_file">
           <property name="text">
            <string>Input File</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_input_file</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_input_file">
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_input_file">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_help">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Batch Mode?</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_input_options" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="5" column="1">
          <widget class="QSpinBox" name="spinBox_input_utm_zone">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <property name="minimum">
            <number>7</number>
           </property>
           <property name="maximum">
            <number>22</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="comboBox_input_coordinates">
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Geographic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cartesian</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UTM</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox_input_reference">
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <property name="currentText">
            <string>WGS84</string>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>NAD83(CSRS)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WGS84</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2020</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2014</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2008</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2005</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF97</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF96</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF94</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF93</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF92</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF91</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF90</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF89</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF88</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="dateEdit_input_epoch">
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <property name="time">
            <time>
             <hour>8</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="maximumDateTime">
            <datetime>
             <hour>7</hour>
             <minute>59</minute>
             <second>59</second>
             <year>9998</year>
             <month>1</month>
             <day>15</day>
            </datetime>
           </property>
           <property name="displayFormat">
            <string notr="true">yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
           <property name="date">
            <date>
             <year>2010</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_input_reference">
           <property name="text">
            <string>Reference Frame</string>
           </property>
           <property name="buddy">
            <cstring>comboBox_input_reference</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_input_utm_zone">
           <property name="text">
            <string>UTM Zone</string>
           </property>
           <property name="buddy">
            <cstring>spinBox_input_utm_zone</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_input_coordinates">
           <property name="text">
            <string>Coordinates</string>
           </property>
           <property name="buddy">
            <cstring>comboBox_input_coordinates</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_input_epoch">
           <property name="text">
            <string>Epoch (YYYY-MM-DD)</string>
           </property>
           <property name="buddy">
            <cstring>dateEdit_input_epoch</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_input_vertical_reference">
           <property name="text">
            <string>Vertical Reference</string>
           </property>
           <property name="buddy">
            <cstring>comboBox_input_vertical_reference</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QComboBox" name="comboBox_input_vertical_reference">
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <item>
            <property name="text">
             <string>WGS84</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_output">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(36, 31, 49)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_output">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Destination</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_output">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_output_file" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_output_file">
           <property name="text">
            <string>Output file</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_output_file</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_output_file">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_output_file">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_epoch_trans">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Epoch Transformation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_output_options" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="7" column="0">
          <widget class="QLabel" name="label_output_vertical_reference">
           <property name="text">
            <string>Vertical Reference</string>
           </property>
           <property name="buddy">
            <cstring>comboBox_output_vertical_reference</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="comboBox_output_reference">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <item>
            <property name="text">
             <string>NAD83(CSRS)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WGS84</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2020</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2014</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2008</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2005</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF2000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF97</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF96</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF94</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF93</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF92</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF91</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF90</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF89</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITRF88</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_output_coordinates">
           <property name="text">
            <string>Coordinates</string>
           </property>
           <property name="buddy">
            <cstring>comboBox_output_coordinates</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_output_reference">
           <property name="text">
            <string>Reference Frame</string>
           </property>
           <property name="buddy">
            <cstring>comboBox_output_reference</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_output_epoch">
           <property name="text">
            <string>Epoch (YYYY-MM-DD)</string>
           </property>
           <property name="buddy">
            <cstring>dateEdit_output_epoch</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="spinBox_output_utm_zone">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <property name="minimum">
            <number>7</number>
           </property>
           <property name="maximum">
            <number>22</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDateEdit" name="dateEdit_output_epoch">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
           </property>
           <property name="keyboardTracking">
            <bool>true</bool>
           </property>
           <property name="time">
            <time>
             <hour>8</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="displayFormat">
            <string notr="true">yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
           <property name="date">
            <date>
             <year>2010</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="comboBox_output_coordinates">
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Geographic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cartesian</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UTM</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_output_utm_zone">
           <property name="text">
            <string>UTM Zone</string>
           </property>
           <property name="buddy">
            <cstring>spinBox_output_utm_zone</cstring>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QComboBox" name="comboBox_output_vertical_reference">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border : 1px solid black;</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>GRS80</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CGVD2013/CGG2013a</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CGVD2013/CGG2013</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CGVD28/HT2_2010v70</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_logs" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(36, 31, 49)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_log_output">
        <property name="text">
         <string>Logs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_log_output">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(31, 26, 40);
border : 1px solid black;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_actions" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;
text-align: center;
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_convert">
           <property name="styleSheet">
            <string notr="true">color: rgb(61, 56, 70);
background-color: rgb(87, 227, 137)
</string>
           </property>
           <property name="text">
            <string>Convert</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_upgrade_link">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/HakaiInstitute/LAS-TRX/releases/tag/0.0.0&quot;&gt;&lt;span style=&quot;text-decoration: underline; color:rgb(153, 193, 241)&quot;&gt;New version available (v0.0.0)&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_input_file</tabstop>
  <tabstop>toolButton_input_file</tabstop>
  <tabstop>comboBox_input_reference</tabstop>
  <tabstop>dateEdit_input_epoch</tabstop>
  <tabstop>comboBox_input_coordinates</tabstop>
  <tabstop>spinBox_input_utm_zone</tabstop>
  <tabstop>comboBox_input_vertical_reference</tabstop>
  <tabstop>lineEdit_output_file</tabstop>
  <tabstop>toolButton_output_file</tabstop>
  <tabstop>comboBox_output_reference</tabstop>
  <tabstop>dateEdit_output_epoch</tabstop>
  <tabstop>comboBox_output_coordinates</tabstop>
  <tabstop>spinBox_output_utm_zone</tabstop>
  <tabstop>comboBox_output_vertical_reference</tabstop>
  <tabstop>pushButton_convert</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
